apps/builder/app/canvas/shared/use-shortcuts.ts

```diff
@@ -40,6 +42,8 @@ export const useShortcuts = () => {
 
   useHotkeys(shortcuts.esc, shortcutHandlerMap.esc, options, []);
 
+  useHotkeys(shortcuts.enter, shortcutHandlerMap.enter, {}, []);
```

Comment:
Don't need shortcuts or shortcutHandlerMap.

```
  useHotkeys("enter", enterEditingMode, {}, []);
```

Reply:
`shortcutHandlerMap[name]();` used below so just to be sure that same handlers are used